libjolie/src/jolie/lang/parse/SemanticVerifier.java

```diff
@@ -386,6 +387,9 @@ public void validate()
 		}
 	}
 
+	@Override
+	public void visit( ReThrowStatement n ) {}
```

Comment:
Mmh yes. Slight variant:

- Throw a warning whenever a procedure that contains a rethrow is called from outside of a fault handler. Notice that being called from a handler for `this` does not count, since that's not a _fault_ handler.
- We need the runtime error anyway, to have a complete implementation of rethrow, so let's do it.

How about that? @jolie/developers this is basically the last bit, feedback?